packages/fxa-settings/src/components/images/lightbulb.svg

```diff
@@ -1,10 +1,10 @@
-<path d="M223.374 36.3736C223.374 39.812 221.166 47.4167 217.632 49.1836C217.19 57.1346 213.655 59.6241 208.356 58.9015C204.348 58.9015 201.288 58.9015 199.963 49.6253C198.278 46.5138 194.662 42.5577 192.896 39.0239C192.896 31.4613 200.793 22.6802 208.356 22.6802C215.918 22.6802 223.374 28.8109 223.374 36.3736Z" fill="#7542E5"/>
+<svg width="68" height="68" viewBox="0 0 68 68" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Unrelated nit, but the 👁️  see what the 👁️  see&hellip;
Can/should we rename this file to "graphic_lightbulb.svg" to be consistent with the other SVGs in the [/fxa-settings/src/components/images/](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings/src/components/images) folder?

Also, just for giggles confirmed the other sizes in the wizard:
- [graphic_key.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_key.svg): `<svg width="71" height="39" …>`
- [graphic_lock.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_lock.svg): `<svg width="53" height="65" …>`
- [graphic_security_shield.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/graphic_security_shield.svg) `<svg width="250" height="200" …>`
- [lightbulb.svg](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/images/lightbulb.svg): `<svg width="68" height="68" viewBox="0 0 68 68" …>` (I think was the only non-minimized one as well)

Reply:
Good idea about the naming, and thanks for catching that it is the only non-minimized one!